File: src/main/java/models/Event.java
package models;

import javafx.beans.property.*;
import javafx.scene.control.TableColumn;
import javafx.scene.control.TableView;

public class Event {
    private  int id ;
    private  String nom_event ;
    private  String categorie ;
    private  String date;
    private  String heure ;
    private  String lieu ;
    private int capacite ;
    private  Float cout ;

    public Event(int id, String nom_event, String categorie, String date, String heure, String lieu, int capacite, Float cout) {
        this.id = id;
        this.nom_event = nom_event;
        this.categorie = categorie;
        this.date = date;
        this.heure = heure;
        this.lieu = lieu;
        this.capacite = capacite;
        this.cout = cout;
    }

    public Event( String nom_event, String categorie, String date, String heure, String lieu, int capacite, Float cout) {

        this.nom_event = nom_event;
        this.categorie = categorie;
        this.date = date;
        this.heure = heure;
        this.lieu = lieu;
        this.capacite = capacite;
        this.cout = cout;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom_event() {
        return nom_event;
    }

    public void setNom_event(String nom_event) {
        this.nom_event = nom_event;
    }

    public String getCategorie() {
        return categorie;
    }

    public void setCategorie(String categorie) {
        this.categorie = categorie;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getHeure() {
        return heure;
    }

    public void setHeure(String heure) {
        this.heure = heure;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public int getCapacite() {
        return capacite;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }

    public Float getCout() {
        return cout;
    }

    public void setCout(Float cout) {
        this.cout = cout;
    }

    @Override
    public String toString() {
        return "Event{" +
                "id=" + id +
                ", nom_event='" + nom_event + '\'' +
                ", categorie='" + categorie + '\'' +
                ", date=" + date +
                ", heure=" + heure +
                ", capacite=" + capacite +
                ", cout=" + cout +
                ", lieu='" + lieu + '\'' +
                '}';
    }
}

File: Desktop/gestion_event/src/Entity/Event.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use DateTimeInterface;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;

/**
 * Event
 *
 * @ORM\Table(name="event")
 * @ORM\Entity(repositoryClass="App\Repository\EventRepository")
 * @UniqueEntity(fields={"nomEvent"}, message="Ce nom d'événement est déjà utilisé.")
 */
class Event
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string|null
     *
     * @ORM\Column(name="nom_event", type="string", length=255, nullable=true)
     */
    private $nomEvent;

    /**
     * @var string|null
     *
     * @ORM\Column(name="categorie", type="string", length=255, nullable=true)
     */
    private $categorie;

    /**
     * @var \DateTimeInterface|null
     *
     * @ORM\Column(name="date", type="date", nullable=true)
     */
    private $date;

    /**
     * @var \DateTimeInterface|null
     *
     * @ORM\Column(name="heure", type="time", nullable=true)
     */
    private $heure;

    /**
     * @var string|null
     *
     * @ORM\Column(name="lieu", type="string", length=255, nullable=true)
     */
    private $lieu;

     /**
     * @var int|null
     *
     * @ORM\Column(name="capacite", type="integer", nullable=true)
     * @Assert\NotNull(message="La capacité ne peut pas être vide.")
     * @Assert\GreaterThan(value=100, message="La capacité doit être supérieure à 100.")
     */
    private $capacite;

  /**
     * @var float|null
     *
     * @ORM\Column(name="cout", type="float", nullable=true)
     * @Assert\NotNull(message="Le coût ne peut pas être vide.")
     * @Assert\GreaterThanOrEqual(value=0, message="Le coût ne peut pas être négatif.")
     */
    private $cout;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomEvent(): ?string
    {
        return $this->nomEvent;
    }

    public function setNomEvent(?string $nomEvent): static
    {
        $this->nomEvent = $nomEvent;

        return $this;
    }

    public function getCategorie(): ?string
    {
        return $this->categorie;
    }

    public function setCategorie(?string $categorie): static
    {
        $this->categorie = $categorie;

        return $this;
    }

    public function getDate(): ?DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(?DateTimeInterface $date): static
    {
        $this->date = $date;

        return $this;
    }

    public function getHeure(): ?DateTimeInterface
    {
        return $this->heure;
    }

    public function setHeure(?DateTimeInterface $heure): static
    {
        $this->heure = $heure;

        return $this;
    }

    public function getLieu(): ?string
    {
        return $this->lieu;
    }

    public function setLieu(?string $lieu): static
    {
        $this->lieu = $lieu;

        return $this;
    }

    public function getCapacite(): ?int
    {
        return $this->capacite;
    }

    public function setCapacite(?int $capacite): static
    {
        $this->capacite = $capacite;

        return $this;
    }

    public function getCout(): ?float
    {
        return $this->cout;
    }

    public function setCout(?float $cout): static
    {
        $this->cout = $cout;

        return $this;
    }
    public function __toString()
    {
        return $this->nomEvent;
    }
}
